greenheart/tools/eco/utilities.py

```diff
@@ -273,139 +277,176 @@ def visualize_plant(
-    rotor_radius = rotor_diameter / 2.0
+        desal_equipment_side = np.sqrt(desal_equipment_area)
 
-    # get turbine tower base diameter
-    tower_base_diameter = wind_cost_outputs.orbit_project.config["turbine"]["tower"][
-        "section_diameters"
-    ][
-        0
-    ]  # in m
-    tower_base_radius = tower_base_diameter / 2.0
-
-    # get turbine locations
-    turbine_x = (
-        wind_cost_outputs.orbit_project.phases["ArraySystemDesign"].turbines_x.flatten()
-        * 1e3
-    )  # ORBIT gives coordinates in km, convert to m
-    turbine_x = turbine_x[~np.isnan(turbine_x)]
-    turbine_y = (
-        wind_cost_outputs.orbit_project.phases["ArraySystemDesign"].turbines_y.flatten()
-        * 1e3
-    )  # ORBIT gives coordinates in km, convert to m
-    turbine_y = turbine_y[~np.isnan(turbine_y)]
-
-    # get offshore substation location and dimensions
-    substation_x = (
-        wind_cost_outputs.orbit_project.phases["ArraySystemDesign"].oss_x * 1e3
-    )  # ORBIT gives coordinates in km, convert to m (treated as center)
-    substation_y = (
-        wind_cost_outputs.orbit_project.phases["ArraySystemDesign"].oss_y * 1e3
-    )  # ORBIT gives coordinates in km, convert to m (treated as center)
-    substation_side_length = 20  # [m] just based on a large substation (https://www.windpowerengineering.com/making-modern-offshore-substation/) since the dimensions are not available in ORBIT
+        
 
-    # set onshore substation dimensions
-    onshore_substation_x_side_length = 127.25  # [m] based on 1 acre area https://www.power-technology.com/features/making-space-for-power-how-much-land-must-renewables-use/
-    onshore_substation_y_side_length = 31.8  # [m] based on 1 acre area https://www.power-technology.com/features/making-space-for-power-how-much-land-must-renewables-use/
+        # get pipe points
+        pipe_x = np.array([substation_x - 1000, substation_x])
+        pipe_y = np.array([substation_y, substation_y])
 
-    # get equipment platform location and dimensions
-    equipment_platform_area = platform_results["toparea_m2"]
-    equipment_platform_side_length = np.sqrt(equipment_platform_area)
-    equipment_platform_x = (
-        substation_x - substation_side_length - equipment_platform_side_length / 2
-    )  # [m] (treated as center)
-    equipment_platform_y = substation_y  # [m] (treated as center)
+        # get cable points
+        cable_x = pipe_x
+        cable_y = pipe_y
 
-    # get platform equipment dimensions
-    if design_scenario["electrolyzer_location"] == "turbine":
-        desal_equipment_area = desal_results[
-            "per_turb_equipment_footprint_m2"
-        ]  # equipment_footprint_m2
-    elif design_scenario["electrolyzer_location"] == "platform":
-        desal_equipment_area = desal_results["equipment_footprint_m2"]
     else:
-        desal_equipment_area = 0
+        turbine_x = np.array(hopp_config["technologies"]["wind"]["floris_config"]["farm"]["layout_x"])
```

Comment:
Perhaps not worth addressing here, but may need to create an issue: if not offshore, will we always be using FLORIS for the wind component? Would we ever use PySAM wind here? Probably not since PySAM doesn't model turbine locations, but perhaps a catch would be good to throw an error if the user tries to visualize the plant with PySAM wind configured?

Reply:
Good point. I added a `NotImplementedError` to the function.